src/api/consumer/lib/tag.ts

```diff
@@ -51,26 +52,31 @@ type TagParams = {
   ids: string[];
   all: boolean;
   snapped: boolean;
-  scope?: string | boolean;
+  workspace?: string | boolean;
   includeImported: boolean;
   incrementBy: number;
 } & BasicTagParams;
 
 export async function tagAction(tagParams: TagParams): Promise<TagResults | null> {
-  const { ids, all, exactVersion, releaseType, force, ignoreIssues, scope, includeImported, persist, snapped } =
+  const { ids, all, exactVersion, releaseType, force, ignoreIssues, workspace, includeImported, persist, snapped } =
     tagParams;
   const idsHasWildcard = hasWildcard(ids);
-  const isAll = Boolean(all || scope || idsHasWildcard);
   const validExactVersion = validateVersion(exactVersion);
-  const preHook = isAll ? PRE_TAG_ALL_HOOK : PRE_TAG_HOOK;
+  const isMany = Boolean(all || workspace || idsHasWildcard);
+  const preHook = isMany ? PRE_TAG_ALL_HOOK : PRE_TAG_HOOK;
   HooksManagerInstance.triggerHook(preHook, tagParams);
   const consumer = await loadConsumer();
+  if (!consumer.isLegacy) {
+    if (force && all) {
+      throw new GeneralError('you can use either "--force" or "--all", but not both. consider using "--workspace"');
```

Comment:
error text suggestion (less confusing):
'"--all" cannot be used with "--force". To force-tag all components in workspace use "--force --workspace"'